Description: Add missing Libs flags
 Debian only ships libopendht.a. The pkg-config file is only
 included in the dev package. This modifies it so that the
 linking works with libopendht.a
Author: Alexandre Viau <aviau@debian.org>

--- a/opendht.pc.in
+++ b/opendht.pc.in
@@ -5,7 +5,7 @@
 Name: OpenDHT
 Description: C++14 Distributed Hash Table library
 Version: @VERSION@
-Libs: -L${libdir} -lopendht
+Libs: -L${libdir} -lopendht -lnettle -lgnutls -largon2 -lhttp_parser
 Libs.private: @http_parser_lib@ -pthread
 Requires.private: gnutls >= 3.3, nettle >= 2.4@argon2_lib@@jsoncpp_lib@@openssl_lib@
 Cflags: -I${includedir}
